CMakeLists.txt

```diff
@@ -1,76 +1,124 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(libyaml_vendor)
+project(libyaml_vendor VERSION 1.0.0)
```

Comment:
This is not needed I think.

Reply:
I think we should indicate the version otherwise this error appears:

```
CMake Error at /usr/share/cmake-3.10/Modules/WriteBasicConfigVersionFile.cmake:40 (message):
  No VERSION specified for WRITE_BASIC_CONFIG_VERSION_FILE()
Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/CMakePackageConfigHelpers.cmake:207 (write_basic_config_version_file)
  CMakeLists.txt:104 (write_basic_package_version_file)
```